templates/events-list.php

```diff
@@ -1,18 +1,30 @@
 <?php
 gp_title( __( 'Translation Events' ) );
 gp_tmpl_header();
-
+gp_tmpl_load( 'events-header', get_defined_vars(), dirname( __FILE__ ) );
 ?>
 
+
 <h2 class="event_page_title">Upcoming Translation Events</h2>
+<div class="event-left-col">
 <?php
-
-
-$_paged = ( get_query_var( 'paged' ) ) ? get_query_var( 'paged' ) : 1;
-$args   = array(
+$current_datetime_utc = ( new DateTime( 'now', new DateTimeZone( 'UTC' ) ) )->format( 'Y-m-d H:i:s' );
+$_paged               = ( get_query_var( 'paged' ) ) ? get_query_var( 'paged' ) : 1;
+$args                 = array(
 	'post_type'      => 'event',
 	'posts_per_page' => 10,
 	'paged'          => $_paged,
+	'post_status'    => 'publish',
+	'meta_query'     => array(
+		array(
+			'key'     => '_event_start',
+			'value'   => $current_datetime_utc,
+			'compare' => '>=',
+			'type'    => 'DATETIME',
+		),
+	),
+	'orderby'        => 'meta_value',
+	'order'          => 'ASC',
 );
 
 $query = new WP_Query( $args );
```

Comment:
I think it would make sense to move the query out of the template and into `WPORG_GP_Translation_Events_Route::events_list()`, so that the templates are logic-less. Looking at the other templates (e.g. `events-edit`), that seems to be the case there, I think it would be good to do that for this template too.

This might not be in scope for this PR though, so we could do it in a follow-up PR.